crates/torii/core/src/engine.rs

```diff
@@ -253,6 +253,10 @@ impl<'db, P: Provider + Sync> Engine<'db, P> {
                 processor
                     .process(&self.world, self.db, block, invoke_receipt, event_id, event)
                     .await?;
+            } else {
+                let keys: Vec<String> = event.keys.iter().map(|k| format!("{:#x}", k)).collect();
+                let data: Vec<String> = event.data.iter().map(|d| format!("{:#x}", d)).collect();
+                trace!("Event {{ keys: {:?}, data: {:?} }}", keys, data);
```

Comment:
Can we use structured logging here?
```suggestion
                trace!("unprocessed event", keys = ?keys, data = ?data);
```

Reply:
We should also start namespacing our loggers but that can be a future task

Reply:
@dubzn let's merge this once you make the structured logging change

Reply:
Hey Tarrence, please check the changes and let me know wdyt. 
Sorry for the delay